src/RFlow/LetterTrain.jsx

```
@@ -6788,8 +6789,8 @@ const LetterTrain = ({
                     >
                       <Box
                         sx={{
-                          minWidth: { xs: 40, sm: 50, md: 60 },
-                          minHeight: { xs: 40, sm: 50, md: 60 },
+                          minWidth: { xs: "clamp(24px, calc(40px * (100vw - 20px) / 350px), 40px)", sm: 50, md: 60 },
```

ESLint

[error] Replace `·xs:·"clamp(24px,·calc(40px·*·(100vw·-·20px)·/·350px),·40px)",·sm:·50,·md:·60` with `⏎····························xs:·"clamp(24px,·calc(40px·*·(100vw·-·20px)·/·350px),·40px)",⏎····························sm:·50,⏎····························md:·60,⏎·························`

(prettier/prettier)